atmos_model.F90

```diff
@@ -155,11 +155,13 @@ module atmos_model_mod
 logical :: dycore_only  = .false.
 logical :: debug        = .false.
 !logical :: debug        = .true.
+logical :: merge_import = .true.
+logical :: debug_merge  = .false.
 logical :: sync         = .false.
 integer, parameter     :: maxhr = 4096
 real, dimension(maxhr) :: fdiag = 0.
 real                   :: fhmax=384.0, fhmaxhf=120.0, fhout=3.0, fhouthf=1.0,avg_max_length=3600.
-namelist /atmos_model_nml/ blocksize, chksum_debug, dycore_only, debug, sync, fdiag, fhmax, fhmaxhf, fhout, fhouthf, ccpp_suite, avg_max_length
+namelist /atmos_model_nml/ blocksize, chksum_debug, dycore_only, debug, sync, fdiag, fhmax, fhmaxhf, fhout, fhouthf, ccpp_suite, avg_max_length, merge_import, debug_merge
```

Comment:
please remove fdiag, fhmax, fhmaxhf, fhout, fhouthf that were removed in PR#691. 
Also why do you need to merge_import variable in dycore atmosphere.F90

Reply:
I'm open to replacing or moving the merge_import configuration option but as long as it's part of the atmos_model_nml namelist it needs to be included anywhere the namelist is read.

Reply:
@junwang-noaa, sure, I will remove fdiag, fhmax, fhmaxhf, fhout, fhouthf accordingly.
@danrosen25, in terms of the merge_import, debug_merge options, I was thinking of they can be moved the the same section as (cplflx, cplwav, etc.), since they are all coupling related. Another advantage is that, in this case, we will not need to make any changes to the GFDL_atmos_cubed_sphere dycore (thus this PR: NOAA-GFDL/GFDL_atmos_cubed_sphere/pull/121 will no longer needed). 

BTW, probably can also consider renaming debug_merge into merge_debug. Thanks!

Reply:
@BinLiu-NOAA @junwang-noaa 
How about I move these configuration options into gfs_physics_nml and rename them `cpl_imp_mrg` and `cpl_imp_dbg`? They will be part of the GFS_control_type and can be referenced using Model%cpl_imp_mrg and Model%cpl_imp_dbg. They will both default to false. See the metadata below.

ccpp/data/GFS_typedefs.meta
```
[cpl_imp_mrg]
  standard_name = flag_for_merging_imported_data
  long_name = flag controlling cpl_imp_mrg for imported data (default off)
  units = flag
  dimensions = ()
  type = logical
[cpl_imp_dbg]
  standard_name = flag_for_debugging_merged_imported_data
  long_name = flag controlling cpl_imp_dbg for imported data (default off)
  units = flag
  dimensions = ()
  type = logical
```
ccpp/data/GFS_typedefs.F90
`NAMELIST /gfs_physics_nml/ `

Reply:
@danrosen25, this looks good to me. And with this, the NOAA-GFDL/GFDL_atmos_cubed_sphere/pull/121 PR will no longer be needed). Thanks!

Reply:
@BinLiu-NOAA @junwang-noaa 
Okay, I'm going forward with the flags/descriptions as displayed below. I'm working on this now.
```
[cpl_imp_mrg]
  standard_name = flag_for_merging_imported_data
  long_name = flag controlling cpl_imp_mrg for imported data (default off)
  units = flag
  dimensions = ()
  type = logical
[cpl_imp_dbg]
  standard_name = flag_for_debugging_merged_imported_data
  long_name = flag controlling cpl_imp_dbg for imported data (default off)
  units = flag
  dimensions = ()
  type = logical
```

Reply:
This comment has been addressed in https://github.com/NOAA-EMC/fv3atm/pull/355/commits/6ac48b1d36b5559500dcb4718bae9140206fc2b8